pkg/console/operator/operator.go

```diff
@@ -122,8 +117,7 @@ func NewConsoleOperator(
 		recorder:       recorder,
 		resourceSyncer: resourceSyncer,
 		// proxy
-		proxyCfgLister:   proxyCfgInformer.Lister(),
-		proxyCfgInformer: proxyCfgInformer.Informer(),
+		proxy: configClient.Proxies(),
```

Comment:
nit: this is a config, so I may just put it at the top with the 3 existing configs, console, operator, infrastructure.

```go 
operatorConfigClient:       operatorConfigClient.Consoles(),
consoleConfigClient:        configClient.Consoles(),
infrastructureConfigClient: configClient.Infrastructures(),
proxyConfigClient: configClient.Proxies(),
```